assay/src/client/AssayDesigner/AssayDesigner.tsx

```diff
@@ -29,11 +29,12 @@ type State = {
     isLoadingModel: boolean,
     hasDesignAssayPerm?: boolean
     message?: string
-    dirty: boolean
 }
 
 export class App extends React.Component<any, State> {
 
+    private _dirty = false;
```

Comment:
Should _dirty be defined of type boolean?
```suggestion
    private _dirty: boolean = false;
```